docs/installing/README.md

```diff
@@ -49,7 +49,7 @@ curl -fsSL https://starship.rs/install.sh | bash -s -- -b /data/data/com.termux/
 nix-env -iA nixos.starship
 ```
 
-#### Declarative, single user, via [home-manager](home-manager)
+#### Declarative, single user, via [home-manager](https://github.com/nix-community/home-manager)
```

Comment:
Linking to the Github repo is fine but other links point either to frontpages or wikis. In this case, we could follow the suit by linking to the manual https://nix-community.github.io/home-manager
@kidonng What do you think?

Reply:
I'm aware of the manual, but it directly jumps to installation and usage without explaining what is it and "Words of warning" like the README on GitHub does. This is friendlier for those who may not be familiar with what the `home-manager` is.

Reply:
> This is friendlier for those who are not familiar with `home-manager.

That is exactly what I wanted to make sure, thanks!